src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        class="card__img" 
+        src="./images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA&sol;A)"
+      >
+
+      <h1 class="card__title">APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h1>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__revievs">
+        <div class="stars">
```

Comment:
Please add additional class _stars--n_ to draw the number of stars depending on the class modifier. (Previous stars task)

Reply:
but 
![Screenshot_1](https://user-images.githubusercontent.com/69647747/224793701-197a054a-d03a-42ab-b439-3afd28c4b499.jpg)